mdpow/workflows/dihedrals.py

```diff
@@ -759,7 +817,7 @@ def automated_dihedral_analysis(dirname=None, df_save_dir=None, figdir=None,
 
```

Comment:
If you actually required `figdir` to be set (but will soon change to an positional arg anyway), add an assertion
```python
assert figdir is not None, "figdir MUST be set, even though it is a kwag. Will be changed with #244"
```